app/react/translations/detail/renderers/BooleanRenderer.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+import React from 'react';
+import { Box } from 'radium-flex';
+
+type PropTypes = {
+  value: string
+};
+
+export default function BooleanRenderer({ value }: PropTypes) {
+  return (
+    <Box>
+      {value}
```

Comment:
values are strings, because they get fetched like strings, no need to convert it, since we need to render strings, not booleans anyway